Mathlib/Algebra/Ring/Associated.lean

```diff
@@ -12,11 +12,18 @@ import Mathlib.Algebra.Ring.Units
 
 assert_not_exists OrderedCommMonoid Multiset Field
 
-namespace Associated
 variable {M : Type*} [Monoid M] [HasDistribNeg M] {a b : M}
+namespace Associated
+
 
 lemma neg_left (h : Associated a b) : Associated (-a) b := let ⟨u, hu⟩ := h; ⟨-u, by simp [hu]⟩
 lemma neg_right (h : Associated a b) : Associated a (-b) := h.symm.neg_left.symm
 lemma neg_neg (h : Associated a b) : Associated (-a) (-b) := h.neg_left.neg_right
 
 end Associated
+
+variable (a)
+
+lemma associated_neg_self_left : Associated (-a) a := Associated.rfl.neg_left
```

Comment:
I don't think we actually want this lemma; for instance, we do not have https://loogle.lean-lang.org/?q=Commute+%28-%3Fa%29+%3Fa

Reply:
If we did have it, it should be `Associated.neg_self_left` or similar to enable dot constructor notation; but I'd like to see some evidence of other lemmas of this shape first.

Reply:
I removed the lemmas